examples/llm/train.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+source /workdisk/abhi/examples/venv-new/bin/activate
+composer main.py yamls/mosaic_gpt/125m.yaml train_loader.dataset.split=train_small
```

Comment:
Is this file supposed to be here? It doesn't seem to be referenced anywhere and assumes there's a `venv-new` that something else creates + cleans up

Reply:
Yup this is my bad, will remove.